bindings/java/src/org/sleuthkit/datamodel/TskCaseDbBridge.java

```diff
@@ -648,6 +655,7 @@ long addUnallocFsBlockFilesParent(long fsObjId, String name) {
             beginTransaction();
             VirtualDirectory dir = caseDb.addVirtualDirectory(fsIdToRootDir.get(fsObjId), name, trans);
             commitTransaction();
+			unallocFileDirs.put(dir.getId(), dir);
```

Comment:
You might consider formatting the file, looks like the indentation is off a bit.